docs/install.md

```diff
@@ -122,6 +122,45 @@ Select the following tabs for the installation description of your target.
 
     is needed. A recommended size is at least a free space of 10 GiB.
 
+    ??? important "Installation of VirtualBox with VirtualBox extension pack ("Guest additions")"
+        If you plan to install the VirtualBox extension pack, some things have to be taken into account to achieve a working time synchronization. In general:
+
+        - VirtualBox without extension pack needs activated time synchronisation from DietPi (systemd service)
+        - VirtualBox with extension pack needs deactivated time synchronisation from DietPi (systemd service) and some additional installation steps
```

Comment:
While the above two cases are clear, it is not precisely correct, as the DietPi systemd-timesyncd implementation is not the only possibility. As I aim to clarify things better within the DietPi-Config menus, let's try to find a precisely correct and still clear statement here. What do you think about:
- The VirtualBox expansion pack contains an own time sync daemon, which keeps the VM system time in sync with the host system time. The time sync mode in `dietpi-config` should hence be switched to `Custom` to avoid conflicts or timeouts at boot.
- VirtualBox without the extension pack requires a network time sync solution, e.g. one of the `dietpi-config` time sync modes that are based on `systemd-timesyncd`.

Reply:
Ok, changed the description.